Document human readable page title (#2793)
commercetools/merchant-center-application-kit: website/src/releases/2022-09/readable-page-title.mdx

---
date: 2022-09-29
title: Human-readable page titles

> description: Introducing more granular and readable document's titles for all Merchant Center pages to help users identifying the context of the page.

Reviewer: > Introducing more granular and readable document titles for all Merchant Center pages to help users identify the page's context.

type: feature
topics:
  - Merchant Center
  - UI Components
---

Custom Applications use a more readable document's title by default, which can be customized. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.

> As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

Reviewer: Custom Applications now use a more readable, customizable document title by default. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

---
date: 2022-09-29

> title: Readable page title component

Reviewer 2: I would mention the feature in general, not only the component

```suggestion
title: Human-readable page titles
```

> description: We now have a component that can be used to overwrite the default document's `<title>`, to provide improved support for a Human-readable page title

Reviewer 2: Short description of the overall feature.

```suggestion
description: Introducing more granular and readable document's titles for all Merchant Center pages to help users identifying the context of the page.
```

type: feature
topics:
  - Merchant Center
  - UI Components
---

> We now have a [`Human-readable page title`](/development/human-readable-page-title) feature describing as detailed as possible,
> the location or context of the page a user is currently on.
> 
> You can now use the [`<ApplicationPageTitle>`](/api-reference/commercetools-frontend-application-shell#applicationpagetitle) component to overwrite the default page title by providing more details about a page by passing an `additionalParts` props,
> an array of strings carrying different descriptions of the features of the page. This provides more readability and context to the page title.

Reviewer 2: Since we already have the documentation pages, we can simply link to them.

```suggestion
Custom Applications use a more readable document's title by default, which can be customized. You can read more about this feature in the [`Human-readable page title`](/development/human-readable-page-title) documentation.

As always, if you have questions or feedback you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

Author: Thank you. 
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/056508e8f1a58425cb3cf3a4196c7af1db312ed7

---
date: 2022-09-29
title: Readable page title component
description: We now have a component that can be used to overwrite the default document's `<title>`, to provide improved support for a Human-readable page title
type: feature
topics:
  - Merchant Center
  - UI Components
---

We now have a `Human-readable page title` feature describing as detailed as possible,
the location or context of the page a user is currently on.

You can now use the [`<ApplicationPageTitle>` component](/api-reference/commercetools-frontend-application-shell#applicationpagetitle) to overwrite the default page title by providing more details about a page by passing an `additionalParts` props,
an array of strings carrying different descriptions of the features of the page. This provides more readability and context to the page title.

> ```js

Reviewer 3: nit: I wouldn't include the code snippet here since we are already providing links to the the detailed information where readers can have the example code within its context.

Author: Thank you for all the feedback.
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2793/commits/7180e3bc3fd593087432a56d0e48262f57c9fe38
